WordPress/Jetpack/Resources/release_notes.txt

```diff
@@ -1,7 +1,10 @@
-Big news, site fans—your My Site screen just got better. You’ll now see two new tabs you can switch between: Site Menu and Home.
+* [*] a11y: VoiceOver has been improved on the Menus view and now announces changes to ordering. [#18155]
+* [*] Notifications list: remove comment Trash swipe action. [#18349]
+* [*] Reader: Fixed a bug that caused cut off content in reader web view [#16106]
+* [*] Site creation: Adds a new screen asking the user the intent of the site [#18367]
+* [**] Block Editor: Quote block: Adds support for V2 behind a feature flag [https://github.com/WordPress/gutenberg/pull/40133]
+* [**] Block Editor: Update "add block" button's style in default editor view [https://github.com/WordPress/gutenberg/pull/39726]
+* [*] Block Editor: Remove banner error notification on upload failure [https://github.com/WordPress/gutenberg/pull/39694]
+* [*] My Site: display site name in My Site screen nav title [#18373]
+* [*] [internal] Site creation: Adds a new screen asking the user the name of the site [#18280]
```

Comment:
I don't this this one should be included (neither in the WP nor in the JP ones), as we don't want our freelance writer to mention anything about this in the public-facing release notes she'll be crafting from that draft… 😅